src/Symfony/Component/Notifier/Bridge/FakeSms/Tests/FakeSmsLoggerTransportTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\FakeSms\Tests;
+
+use Psr\Log\LoggerInterface;
+use Psr\Log\Test\TestLogger;
+use Symfony\Component\Notifier\Bridge\FakeSms\FakeSmsLoggerTransport;
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Test\TransportTestCase;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+final class FakeSmsLoggerTransportTest extends TransportTestCase
+{
+    public function createTransport(HttpClientInterface $client = null, LoggerInterface $logger = null): TransportInterface
+    {
+        $transport = (new FakeSmsLoggerTransport($logger ?? $this->createMock(LoggerInterface::class), $client ?? $this->createMock(HttpClientInterface::class)));
+
+        return $transport;
+    }
+
+    public function toStringProvider(): iterable
+    {
+        yield ['fakesms+logger://default', $this->createTransport()];
+    }
+
+    public function supportedMessagesProvider(): iterable
+    {
+        yield [new SmsMessage('0611223344', 'Hello!')];
+        yield [new SmsMessage('+33611223344', 'Hello!')];
+    }
+
+    public function unsupportedMessagesProvider(): iterable
+    {
+        yield [new ChatMessage('Hello!')];
+        yield [$this->createMock(MessageInterface::class)];
+    }
+
+    public function testSendWithDefaultTransport()
+    {
+        $message = new SmsMessage($phone = '0611223344', 'Hello!');
+
+        $logger = new TestLogger();
```

Comment:
this class has been removed via https://github.com/php-fig/log/pull/76/files#diff-485f594e2e39a73ccdb1dddb77212303cc1862bb86b5e4f8c4a29b6b734e5371

spotted in https://github.com/symfony/symfony/pull/42123/checks?check_run_id=3076335813

hum

Reply:
I just looked at php-fig/log-util#2 and cannot say if and when this will be merged and tagged.

I would use the solution from @Cerell:
```php
$this->trackingLogger = new class() implements LoggerInterface {
    use LoggerTrait;
    public array $records = [];
    public function log($level, $message, array $context = []): void
    {
        $this->records[] = [
            'level' => $level,
            'message' => $message,
            'context' => $context
        ];
    }
};
```

@Nyholm @derrabus WDYT?

Reply:
In other parts of Symfony we do:
```php
final class TestLogger extends AbstractLogger
{
    public $records = [];

    public function log($level, $message, array $context = []): void
    {
        $this->records[] = [
            'level' => $level,
            'message' => $message,
            'context' => $context,
        ];
    }
}
```

Reply:
I will mimic/backport this part oki